src/main/java/com/google/cloud/genomics/dataflow/utils/ShardedBAMWritingOptions.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloud.genomics.dataflow.utils;
```

Comment:
I actually think it would be good to leave these options in the pipeline file.  I'm starting to think that every file in the pipelines directory should embed their own options.  (which is what I plan to do when I refactor to get rid of GenomicsDatasetOptions per https://github.com/googlegenomics/dataflow-java/issues/114; but I'm interested to hear other opinions)

test/java/com/google/cloud/genomics/dataflow/functions